localstack/utils/cloudformation/template_deployer.py

```diff
@@ -516,6 +519,52 @@ def get_apigw_resource_params(params, **kwargs):
                 'InstanceIds': lambda params, **kw: [kw['resources'][kw['resource_id']]['PhysicalResourceId']]
             }
         }
+    },
+    'EC2::VPC': {
+        'create': {
+            'function': 'create_vpc',
+            'parameters': {
+                'CidrBlock': 'CidrBlock'
+            }
+        },
+        'delete': {
+            'function': 'delete_vpc',
+            'parameters': {
+                'VpcId': 'PhysicalResourceId'
+            }
+        }
+    },
+    'EC2::RouteTable': {
```

Comment:
Let's better move these new deploy templates into `get_deploy_templates(...)` methods in the respective model classes `EC2RouteTable` and `EC2Route` (to be created) in `service_models.py`. 

The goal is that over time we can encapsulate and decentralize the deployment logic inside the model classes directly (to avoid having the centralized code and there large dicts here).